apps/client/src/features/modals/upload-modal/preview/PreviewTable.module.scss

```diff
@@ -0,0 +1,71 @@
+@use "../../../../theme/_ontimeColours" as *;
+
+.container {
+  max-width: 100%;
+  max-height: max(300px, 30vh);
+  overflow: scroll;
+}
+
+.rundownPreview {
+  font-size: calc(1rem - 2px);
+  overflow-x: scroll;
+}
+
+.header,
+.body {
+  th {
+    font-weight: 400;
+    height: unset;
+    line-height: calc(1rem - 2px);
+    white-space: nowrap;
+    padding-left: 0.25rem;
+    padding-right: 1rem;
+  }
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  background-color: white;
+  box-shadow: 0 2px $gray-50;
+  z-index: 3;
+
+  th {
+    font-weight: 200;
+    text-align: left;
+  }
+
+  tr {
+    word-wrap: unset;
+  }
+}
+
+.body {
+  td {
+    text-align: left;
```

Comment:
the preview could use some line sepperation
![image](https://github.com/cpvalente/ontime/assets/19875125/f2cc0254-37f1-4069-af8f-82d246f34dfc)
|>
![image](https://github.com/cpvalente/ontime/assets/19875125/30f94833-91b8-4204-9b36-11e62967e4f1)